Performance-Task/README.md

```diff
@@ -0,0 +1,35 @@
+# The Project
+
+## The Brief
+
+Our client wants to build up a subscriber list to market their new product. The client is paying for AWS and wants us to use it for this project. You are a full-stack engineer and your project manager has tasked you with finding a way to store all the subscriber's emails and creating a way to capture subscriber emails.
+
+The frontend team has come up with a couple of prototypes for the frontend form. You could either wire up one of their prototypes or roll your own!
+
+Your project manager recommends you use the command `cdk init sample-app --language=[csharp|fsharp|java|javascript|python|typescript]` to save time. AWS CDK converts into JSON/YAML so the language you implement this feature is completely up to you!
+
+## Requirements:
+
+- Wire up a form for users to input their email
+- Handle user input with AWS Lambda
+- Store that email on AWS using DynamoDB
```

Comment:
Low context bullet points to get the learner thinking in the right direction for the task

Reply:
I would convert these into checkboxes (they are just much inviting and we are talking about required steps here). Also, it might be a good idea, to add additional requirements or suggestions for more eager viewers. 🤔 (add an option to delete emails, display emails, add the company logo to the form etc)